docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -22,13 +25,17 @@ DataStore.configure({
 
 When DataStore starts syncing, only Posts with `rating > 5` and Comments with `status === 'active'` will be synced down to the user's local store.
 
+<amplify-callout>
```

Comment:
Do we want this to be a warning callout or a general one?

Reply:
I don't think it needs to be a warning since we handle it for the customer if they pass > 1. So it's non-breaking if they don't follow it. The one on line 95 probably makes more sense as a warning though